src/postman/geoserver-build.postman_collection.json

```diff
@@ -0,0 +1,1385 @@
+									"raw": "http://{{host}}{{site}}/rest/services/wfs/transforms",
+									"protocol": "http",
+									"host": [
+										"{{host}}{{site}}"
+									],
+									"path": [
+										"rest",
+										"services",
+										"wfs",
+										"transforms"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "WFS marvl xml",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Status code is 200\", function () {",
+											"    pm.response.to.have.status(200);",
+											"});",
+											"",
+											"pm.test(\"Body has valid data\", function () {",
+											"    var jsonData = xml2Json(responseBody);",
+											"    jsonData[\"wfs:FeatureCollection\"][\"gml:featureMember\"].forEach( fm => {",
+											"        pm.expect(fm[\"imos:gsla_fv02_dm_timeseries_url\"][\"imos:file_url\"]).to.include(\"https://imos-data.s3-ap-southeast-2.amazonaws.com\");",
+											"    });",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"auth": {
+									"type": "noauth"
+								},
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "http://{{host}}{{site}}/imos/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=imos%3Agsla_fv02_dm_timeseries_url&maxFeatures=50&outputFormat=marvl%20xml",
+									"protocol": "http",
+									"host": [
+										"{{host}}{{site}}"
+									],
+									"path": [
+										"imos",
+										"ows"
+									],
+									"query": [
+										{
+											"key": "service",
+											"value": "WFS"
+										},
+										{
+											"key": "version",
+											"value": "1.0.0"
+										},
+										{
+											"key": "request",
+											"value": "GetFeature"
+										},
+										{
+											"key": "typeName",
+											"value": "imos%3Agsla_fv02_dm_timeseries_url"
+										},
+										{
+											"key": "maxFeatures",
+											"value": "50"
+										},
+										{
+											"key": "outputFormat",
+											"value": "marvl%20xml"
+										}
+									]
+								}
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "GWC S3 Blob Store",
+					"item": [
+						{
+							"name": "layers",
+							"request": {
+								"method": "GET",
+								"header": []
+							},
+							"response": []
+						},
+						{
+							"name": "layer",
+							"request": {
+								"method": "GET",
+								"header": []
+							},
+							"response": []
+						},
+						{
+							"name": "download image",
+							"request": {
+								"auth": {
+									"type": "noauth"
+								},
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "https://geoserver223-static-cache.s3.ap-southeast-2.amazonaws.com/LayerGroupInfoImpl--62a9b58b%3A144dcb81441%3A-8000/EPSG%3A4326/png/default/2/0/0.png.png",
```

Comment:
Hi @craigrose this being hard coded into the test, maybe need to be more generic?